Python/tests/test_runner_datetime.py

```diff
@@ -83,104 +145,149 @@ def test_datetime_model(culture, model, options, context, source, expected_resul
         simple_parser_assert(actual, expected, 'parent_text', 'ParentText')
         simple_parser_assert(actual, expected, 'start', 'Start')
         simple_parser_assert(actual, expected, 'end', 'End')
-        assert len(actual.resolution['values']) == len(expected['Resolution']['values'])
-        for actual_resilution_value, expected_resoulution_value in zip(actual.resolution['values'], expected['Resolution']['values']):
-            assert_model_resolution(actual_resilution_value, expected_resoulution_value)
+        assert len(
+            actual.resolution['values']) == len(
+            expected['Resolution']['values'])
+        for actual_resilution_value in actual.resolution['values']:
+            assert_model_resolution(
+                actual_resilution_value,
```

Comment:
Can we cause any breaking changes if we fix these typos? (resilution, resoulution)

Reply:
Not really, good catch! I've changed it.